tests/foreman/api/test_permission.py

```diff
@@ -34,6 +35,14 @@ def setUpClass(cls):
             cls.permissions[None].remove('view_log_viewer')
             cls.permissions[None].remove('view_search')
 
+        result = ssh.command('rpm -qa | grep rubygem-foreman_openscap')
+        if result.return_code != 0:
+            cls.permissions.pop('ForemanOpenscap::Policy')
+            cls.permissions.pop('ForemanOpenscap::ScapContent')
+            cls.permissions[None].remove('destroy_arf_reports')
```

Comment:
Yes, Python allows any object being the key of a dictionary and we use None to indicate that a permission is not related to any specific resource. We can call that the way we map global permissions.